### DISCUSSION TITLE
Input file for running RRTMGP baseline example

### DISCUSSION THREAD

**Discussion by nataraj2** [E3SM-Project/E3SM#6871] [EAMxx help] [answered]
I am trying to run [this test](https://github.com/E3SM-Project/E3SM/blob/master/components/eamxx/src/physics/rrtmgp/tests/generate_baseline.cpp) to generate a baseline for the RRTMGP test case, but it requires an input file (probably a `.nc` netcdf file?). Is that there in the repo or can it be generated with some script? Thanks.

**Comment by mahf708** [comment]
This [link](https://web.lcrc.anl.gov/public/e3sm/inputdata/atm/scream/init/rrtmgp-allsky.nc) has the file in question. Let us know if something isn't working.

**Reply by nataraj2** [reply, marked as answer]
That worked. Was able to run the rrtmgp tests successfully. Thanks @mahf708 !
